lms/services/lti_names_roles.py

```diff
@@ -0,0 +1,59 @@
+"""
+Service to talk to the Name and Roles LTIA API.
```

Comment:
It might be good to say why. Especially as we don't use this code.

If I found this, and found out nothing used it, I'd delete it.